oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -166,6 +170,24 @@ public static GoogleCredentials fromStream(InputStream credentialsStream,
   }
 
   /**
+   * Saves the end user credentials in specified file
+   *
+   * @param credentials InputStream containing user credentials in JSON format
+   * @param credentialsFileName Name of file where to store the credentials
+   * @throws IOException An error saving the credentials.
+   */
+  public void saveCredentialsToFile(InputStream credentials, String credentialsFileName) throws IOException {
```

Comment:
It's odd to have a member function (which contains credentials) which also takes distinct credentials as an argument, and doesn't use any member data. The implementation in fact seems generic enough that you could just use other libraries for this, e.g. `FileUtils.copyInputStreamToFile` from Commons IO.

Reply:
I created the method in ```UserCredentials.java``` called ```saveUserCredentials``` which calls the generic method of it's super class called ```saveCredentialsToFile``` to save them into file.
I fixed the implementation and used ```FileUtils.copyInputStreamToFile``` in order to do this.

Reply:
Sorry for guiding you in the wrong direction by having you add a dependency on Commons IO, then asking you to remove it...

In any case, what I meant was that you don't need to have this utility function at all, and you could just put the implementation at the calling site (for you, `UserCredentials.saveUserCredentials`)